src/pages/nfts/profile/[accountAddress].tsx

```diff
@@ -14,9 +14,9 @@ const NftProfilePage = () => {
   const isConnectedProfile = account?.toLowerCase() === accountAddress?.toLowerCase()
   const {
     profile: profileHookState,
-    isFetching: isProfileFetching,
+    isValidating: isProfileFetching,
```

Comment:
We have to use isValidating directly here, since isFetching always comes false after first fetch

Reply:
Not sure if I understand it correctly. You use `isValidating` to trigger refetch on `useNftsForAddress` ?

Reply:
It is not about triggering fetch actually, if we use isFetching that comes always true after inital fetch, when revalidating usernfts it will fetch twice (first with old profile data, second with after profile data is fetched again)